test/new-e2e/tests/windows/install-test/assert.go

```diff
@@ -92,3 +93,40 @@ func AssertGroupMembership(t *testing.T, host *components.RemoteHost, user strin
 	}
 	return true
 }
+
+// RequireAgentRunningWithNoErrors checks the agent is running with no errors
+func RequireAgentRunningWithNoErrors(t *testing.T, client *common.TestClient) {
+	common.CheckAgentBehaviour(t, client)
+}
+
+// getExpectedSignedFilesForAgentMajorVersion returns the list of files that should be signed for the given agent major version
+// as relative paths from the agent install directory.
+func getExpectedSignedFilesForAgentMajorVersion(majorVersion string) []string {
+	paths := []string{
+		// user binaries
+		`bin\agent.exe`,
+		`bin\libdatadog-agent-three.dll`,
+		`bin\agent\trace-agent.exe`,
+		`bin\agent\process-agent.exe`,
+		`bin\agent\system-probe.exe`,
+		// drivers
+		`bin\agent\driver\ddnpm.sys`,
```

Comment:
Once 7.52 is released, we'll need to expand this list to include the security agent and process driver. Could you make a Jira card for that just so we don't forget?

Additionally, I haven't had a chance to look through the entirety of the tests yet, but is there a place were we verify that only the binaries we expect are in the `\agent\` and `\driver\` directory? That could be a way to:
1. Prevent us from not realizing a team added a new binary without notifying / updating the test
2. Make sure no unexpected binaries snuck into a build / release

What do you think?

Reply:
https://datadoghq.atlassian.net/browse/WINA-687
https://datadoghq.atlassian.net/browse/WINA-688
https://datadoghq.atlassian.net/browse/WINA-689